### DISCUSSION TITLE
Where are the chapter challenges located?

### DISCUSSION THREAD

**Discussion by infinite969** [Cyfrin/foundry-full-course-cu#623] [Q&A]
Hello friends, I am seeing many of you asking for help with the chapter challenges for this course. Can someone tell me where these challenges are located? Still kinda new here

**Comment by 0xusmanf** [comment]
At the end of every lesson there are 2 QR codes displayed for Sepolia and Arbitrium network, Scan the QR code and it will take you to the challenge contract. Alternatively [Here](https://github.com/Cyfrin/foundry-full-course-f23/discussions/135#discussion-5326319) are the links to each challenge on both, Sepolia and Arbitrium networks.

**Reply by infinite969** [reply]
Thank you!
